build-system/SERVING.md

```diff
@@ -18,17 +18,18 @@ limitations under the License.
 
 ## Requirements
 - git
-- node + npm
+- node 6+
 - gulp (installed globally)
 - java 8
+- yarn 1.0.2+ (see https://yarnpkg.com/)
```

Comment:
nit: Move this up the list, just below `node`, to reflect the order in which they are likely to be installed.

Reply:
Done